bin/picca_deltas.py

```diff
@@ -296,27 +296,29 @@ def cont_fit(data):
         log.write("Found {} DLAs in forests\n".format(nb_dla_in_forest))
 
     ## cuts
-    for p in list(data.keys()):
+    log.write("INFO: Sample has {} forests\n".format(sp.sum([len(p) for p in data.values()])))
```

Comment:
Maybe something like "Input sample has..."

Reply:
Maybe save the number to a variable so that subsequent similar lines can say "... {} remaining forests (of {} initial)" or something similar

Reply:
Good idea, let's keep that in mind for when we have the time.